source/2019-08-15-octane-release-plan.md

```diff
@@ -0,0 +1,249 @@
+
+One final thing: if you find yourself writing the same `did-insert` code in multiple places, you can extract the code into a *custom modifier*, which can even be packaged into addons. One cool example of a *custom modifier* that's already on npm is `[ember-did-resize-modifer](https://github.com/gmurphey/ember-did-resize-modifier)`, which allows you to attach custom logic to an element, whenever it changes size.
+
+```handlebars
+<div {{did-resize this.onResize}}>
+	Resize the window to see the modifier in action
+</div>
+```
+
+Custom modifiers provide a much more compositional way to package up DOM behavior than mixins, which is the idiomatic approach in classic Ember.
+
+### The `{{on}}` Modifier
+
+In classic Ember, you can add event handlers to the root element of your component by adding methods to your class. For example, to handle clicks on your component's root element, you would add a `click` method to your class. To handle events on other methods you would use the `{{action}}` modifier where you want to handle the event, and nest the method inside of an `actions` hash.
+
+When using Glimmer components, you can handle events on any element using a new `{{on}}` modifier. Since Glimmer components do not have a special root element, `{{on}}` works anywhere.
+
+```handlebars
+<h1 {{on "click" this.toggleBody}}>Hello world</h1>
+
+{{#if this.showingBody}}
+	<div>{{yield}}</div>
+{{/if}}
+```
+
+For more details, see the [API docs for on](https://api.emberjs.com/ember/3.12/classes/Ember.Templates.helpers/methods/on?anchor=on).
+
+### Required `{{this}}` in Templates
+
+In classic Ember, you can refer to properties on a component as `{{propertyName}}`. This was ambiguous with helpers and components, and was deprecated in [RFC 308](https://emberjs.github.io/rfcs/0308-deprecate-property-lookup-fallback.html).
+
+No matter what kind of component you're using, you should start using `this` to refer to component properties in new code. The `[no-implicit-this](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-implicit-this.md)` template lint can help you avoid using this deprecated pattern by accident.
+
+Ember 3.14 will finalize a codemod that uses dynamic information from booting up your app to automatically insert `this` where needed. We recommend waiting for this codemod to be finalized before attempting to mass-migrate your codebase. 
+
+### Angle Bracket Invocation
+
+This one landed a while ago, and many people have already started to use it in their apps. In classic Ember, you would idiomatically invoke components using curly bracket syntax:
+
+```handlebars
+{{input value=this.name}}
+{{#link-to route=this.routeName}}Some content{{/link-to}}
+```
+
+Octane introduces a new way of idiomatically invoking components using curly braces and `@names`.
+
+```handlebars
+<Input @value={{this.name}} />
+<LinkTo @route={{this.routeName}}>Some content<LinkTo>
```

Comment:
Missing slash on the closing `</LinkTo>` tag

Reply:
good catch, thanks!